x/programs/examples/util.go

```diff
@@ -37,6 +70,12 @@ func newKey() (ed25519.PrivateKey, ed25519.PublicKey, error) {
 	return priv, priv.PublicKey(), nil
 }
 
+func fixedByteKey(key ed25519.PublicKey) [32]byte {
```

Comment:
ahh, i actually wrote a test and it turned out this wasn't necessary because `key` is fixed length. I think i was running into this issue earlier because I was converting to a dynamic array somewhere in earlier code. Anyways, this isn't needed anymore but still good to note that borsh serializes fixed/dynamic array differently.